schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE DATABASE doingsdone
+  DEFAULT CHARACTER SET utf8mb4
+  DEFAULT COLLATE utf8mb4_general_ci;
+
+USE doingsdone;
+
+CREATE TABLE users (
+  id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+  date_reg TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+  email VARCHAR(128) UNIQUE NOT NULL,
```

Comment:
In addition to restrictions on syntax, there is a length limit on email addresses. That limit is a maximum of 64 characters (octets) in the "local part" (before the "@") and a maximum of 255 characters (octets) in the domain part (after the "@") for a total length of 320 characters

https://www.rfc-editor.org/errata/eid1690